src/reducers/stages/day.js

```diff
@@ -0,0 +1,470 @@
+import { nextNightState } from './night'
```

Comment:
sorry, I shouldn't have added this file here. It's not working, it's just night.js copied, so don't bother to read it.